proxy-server/proxy/config.go

```diff
@@ -118,17 +118,17 @@ func (hc *HTTPConfig) Watch(name, namespace string, client *kubernetes.Clientset
 	})
 }
 
-// SessionConfig stores the set of user IDs that are actively logged in
+// SessionConfig stores the set of active user session IDs
```

Comment:
Realizing I'm calling this "session ID" here, but it's actually the "session token" in Atlas. Will update to match.

Reply:
Good call